templates/vue/src/components/modals/NodeModal/forms/DyadForm.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <div>
+    <b-form-group>
+      <b-form-checkbox v-model="node.isDyad">
+        Enable dyad node behaviour
+      </b-form-checkbox>
+    </b-form-group>
+    <small id="emailHelp" class="form-text text-muted">
+      When checked, you will be able to see your child's progress in this node. Only
+      the child's progress will count towards this node's completion.
+    </small>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "conditions-form",
+  props: {
+    node: {
+      type: Object,
+      required: true,
+    },
+  },
+}
+</script>
```

Comment:
Might be better to move this directly into the ConditionsForm.vue itself since it is a small component

Reply:
This was also what I considered, but since the ConditionsForm only deals with managing node access conditions, and Dyad node doesn't really fit into that purpose for that class, I thought to create a new component for it for now...